src/pj_transform.c

```diff
@@ -206,12 +207,14 @@ static int geographic_to_projected (PJ *P, long n, int dist, double *x, double *
             projected_loc = pj_fwd3d( geodetic_loc, P);
             if( P->ctx->last_errno != 0 )
             {
-                if( (P->ctx->last_errno != 33 /*EDOM*/
-                        && P->ctx->last_errno != 34 /*ERANGE*/ )
+                if( (P->ctx->last_errno != EDOM /* 33 */
```

Comment:
Isn't it somewhat pointless to keep the errno values as comments?

Reply:
Yes, having them isn't useful.  I wasn't brave enough to remove them.  Will do it now.